docs/3.api/1.composables/use-async-data.md

```diff
@@ -26,7 +26,7 @@ type AsyncDataOptions<DataT> = {
   server?: boolean
   lazy?: boolean
   immediate?: boolean
-  shallow?: boolean
+  deep?: boolean
```

Comment:
Isn't it easier to understand with `shallow` or `shallowRef` instead of `deep` ?

Reply:
I was thinking:
1. it mirrors the `deep` option for `watch`
2. it will make more sense later on if we choose to default to false (so `{ deep: true }` would be required to turn on deep reactivity)